api_review/RadialGradientBrush/RadialGradientBrush.md

```diff
@@ -0,0 +1,184 @@
+
+When `MappingMode` is set to `Absolute`, the X and Y values of the three properties are treated as absolute coordinates within the brush's rendered bounds.
+
+## Windows 10 Version Support
+
+Gradient rendering is supported on Windows 10 version 1903 (v10.0.18362.0) and higher. On previous OS versions the brush will render a solid color specified by the `FallbackColor` property.
+
+## See Also
+
+* [LinearGradientBrush](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Media.LinearGradientBrush)
+
+* [CompositionRadialGradientBrush](https://docs.microsoft.com/uwp/api/windows.ui.composition.compositionradialgradientbrush)
+
+# API Notes
+
+## Fields
+
+| | |
+| - | - |
+| EllipseCenter | The center of the ellipse that contains the gradient. The default is `(0.5, 0.5)`. |
+| EllipseRadius | The radius of the ellipse that contains the gradient. The default is `(0.5, 0.5)`. |
+| GradientOriginOffset | The gradient origin's offset from the center of the element. |
+| GradientStops | A collection of [GradientStop](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.gradientstop) objects that define the gradient. |
+| InterpolationSpace | The color space used to interpolate the gradient's colors. The default is `Auto`. For supported values, see [CompositionRadialGradientBrush.InterpolationSpace](https://docs.microsoft.com/uwp/api/windows.ui.composition.compositiongradientbrush.interpolationspace#Windows_UI_Composition_CompositionGradientBrush_InterpolationSpace) |
+| MappingMode | Defines whether `EllipseCenter`, `EllipseRadius` and `GradientOriginOffset` represent relative coordinates in the range 0 to 1 or absolute coordinates. The default is `RelativeToBoundingBox`. |
+| SpreadMethod | Gets or sets the type of spread method that specifies how to draw a gradient that starts or ends inside the bounds of the object to be painted. The default is `Pad`. |
+
+# API Details
+
+```C++
+[webhosthidden]
+[contentproperty("GradientStops")]
+unsealed runtimeclass RadialGradientBrush : Windows.UI.Xaml.Media.XamlCompositionBrushBase
+{
+    RadialGradientBrush();
+
+    Windows.Foundation.Point EllipseCenter { get; set; };
+
+    Windows.Foundation.Point EllipseRadius { get; set; };
+
+    Windows.Foundation.Point GradientOriginOffset { get; set; };
+
+    Windows.UI.Xaml.Media.BrushMappingMode MappingMode { get; set; };
+
+    Windows.UI.Composition.CompositionColorSpace InterpolationSpace { get; set; };
+
+    Windows.UI.Xaml.Media.GradientSpreadMethod SpreadMethod { get; set; };
+
+    Windows.Foundation.Collections.IObservableVector<Windows.UI.Xaml.Media.GradientStop> GradientStops { get; };
```

Comment:
Discussion:
Note that this is a get-only IVector property, whereas the existing [GradientBrush.GradientStops](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Media.GradientBrush.GradientStops) is a read/write concrete type. The advantage of a concrete type is that it can be created in markup and can be data bound and shared.

But with a read-only property you can still set the stops in markup, you just can't for example define a stop collection in a ResourceDictionary and then use it in multiple places on the page.

The existing GradientStopCollection doesn't raise change notifications (no public API), so implementing this would be difficult unless we created a new collection type.

The read-only property is less work and looks good enough in terms of scenarios we can think of. We can always add a setter later.